Add a metaTitle and a metaDescription to every doc pages (#233)
strapi/documentation: docs/user-docs/latest/content-types-builder/managing-content-types.md

---
title: Managing Content-types - Strapi User Guide

> description: The Content-Types Builder allows to manage any existing content-type or component, even if it is already being used in the Content Manager. They can only be managed one at a time.

Reviewer: ```suggestion
description: Instructions to manage content types and components with the Content-Types Builder.
```
